pkg/quota/v1/evaluator/core/services.go

```diff
@@ -19,31 +19,31 @@ package core
 import (
 	"fmt"
 
-	"k8s.io/api/core/v1"
+	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apiserver/pkg/admission"
 	api "k8s.io/kubernetes/pkg/apis/core"
 	k8s_api_v1 "k8s.io/kubernetes/pkg/apis/core/v1"
-	"k8s.io/kubernetes/pkg/quota"
-	"k8s.io/kubernetes/pkg/quota/generic"
+	"k8s.io/kubernetes/pkg/quota/v1"
+	"k8s.io/kubernetes/pkg/quota/v1/generic"
 )
 
 // the name used for object count quota
-var serviceObjectCountName = generic.ObjectCountQuotaResourceNameFor(v1.SchemeGroupVersion.WithResource("services").GroupResource())
+var serviceObjectCountName = generic.ObjectCountQuotaResourceNameFor(corev1.SchemeGroupVersion.WithResource("services").GroupResource())
 
 // serviceResources are the set of resources managed by quota associated with services.
-var serviceResources = []api.ResourceName{
+var serviceResources = []corev1.ResourceName{
 	serviceObjectCountName,
-	api.ResourceServices,
-	api.ResourceServicesNodePorts,
-	api.ResourceServicesLoadBalancers,
+	corev1.ResourceServices,
+	corev1.ResourceServicesNodePorts,
+	corev1.ResourceServicesLoadBalancers,
 }
 
 // NewServiceEvaluator returns an evaluator that can evaluate services.
 func NewServiceEvaluator(f quota.ListerForResourceFunc) quota.Evaluator {
```

Comment:
> quota.ListerForResourceFunc

It has been renamed from "k8s.io/kubernetes/pkg/quota" to "k8s.io/kubernetes/pkg/quota/v1".
I'm really surprised to see this package name, `quota`, still work here w/o declaring a package name alias!
@yue9944882 any idea how this work?

Reply:
>  "k8s.io/kubernetes/pkg/quota/v1".

b/c we declared `package quota` in the source under `"k8s.io/kubernetes/pkg/quota/v1"`, we can call this an oversight when copy/pasting the quota codes.

Reply:
is that bothering you?